app/routes/__auth/login.ts

```diff
@@ -13,23 +13,23 @@ import { generators } from 'openid-client'
 import { getOpenIDClient, oidcStorage, storage } from './auth.server'
 import type { getUser } from './user.server'
 
-function userFromTokenSet(
+export function userFromTokenSet(
   tokenSet: TokenSet
 ): NonNullable<Awaited<ReturnType<typeof getUser>>> {
   const claims = tokenSet.claims()
   const sub = claims.sub
   const email = claims.email as string
-
+  const refreshToken = tokenSet.refresh_token as string
   const idp =
     claims.identities instanceof Array && claims.identities.length > 0
       ? (claims.identities[0].providerName as string)
       : null
-
+  const cognitoUserName = claims['cognito:username'] as string
   const groups = ((claims['cognito:groups'] ?? []) as string[]).filter(
-    (group) => group.startsWith('gcn.nasa.gov')
+    (group) => group.split('/')[0] == 'gcn.nasa.gov'
```

Comment:
Equivalent, but simpler:

```suggestion
    (group) => group.startsWith('gcn.nasa.gov/')
```

Reply:
Does this change comply with githubs security checks? I had previously used a startswith, but the automated checks were failing. Does the slash at the end get around this?

Reply:
Doesn't matter, it's a false alarm. We aren't using this as a URL. I had previously dismissed the alert.

Reply:
Good enough for me then!